libs/database/src/index/collab_embeddings_ops.rs

```diff
@@ -77,34 +77,35 @@ pub async fn upsert_collab_embeddings(
     .await?;
   }
 
-  for r in records {
-    sqlx::query(
-      r#"INSERT INTO af_collab_embeddings (fragment_id, oid, partition_key, content_type, content, embedding, indexed_at)
+  if !records.is_empty() {
+    // replace existing collab embeddings
+    remove_collab_embeddings(tx, &records[0].object_id).await?;
+    for r in records {
```

Comment:
Should we use [Bulk Inserts](https://www.postgresql.org/docs/current/functions-array.html#FUNCTIONS-ARRAY-TABLE) here?

Reply:
Yes, we could. We could also do it in another PR.